### ISSUE TITLE
Actionability of duplicate id check

### ISSUE THREAD

**Issue by connorjclark** [dequelabs/axe-core#2062]
The Lighthouse team got to discussing the duplicate id checks in axe. We've had a number of reports where users think the check is mistaken, when in fact it was correct.

https://github.com/GoogleChrome/lighthouse/issues/4313
https://github.com/GoogleChrome/lighthouse/issues/4833
https://github.com/GoogleChrome/lighthouse/issues/5561
https://github.com/GoogleChrome/lighthouse/issues/6489
https://github.com/GoogleChrome/lighthouse/issues/8399
https://github.com/GoogleChrome/lighthouse/issues/10374

Found one from axe too: https://github.com/dequelabs/axe-core/issues/294

We were thinking it'd be great if axe provided references to _two_ nodes using the same ID, instead of just one. This would make it much more obvious for the user where the duplication is (and that the tool is not wrong).

**Comment by straker** [comment]
Wow, that's quite the list. I believe we are already doing this using the `relatedNodes` property of the failed check. Is that not the case?

**Comment by connorjclark** [comment]
Ah, we mistakenly thought that we were using `duplicate-id`. We actually are using `duplicate-id-active` and `duplicate-id-aria`, which don't seem to have a `relatedNodes` property.

**Comment by straker** [comment]
`duplicate-id-active` and `duplicate-id-aria` should all have `relatedNodes` properties. All three rules use the same check code.

Testing on a simple page of just

```html
<button id="fail">hi</button>
<button id="fail">hello</button>
```

I get back a related node (inside the `any` array). 

```json
"violations": [
    {
      "id": "duplicate-id-active",
      "impact": "serious",
      "tags": [
        "cat.parsing",
        "wcag2a",
        "wcag411"
      ],
      "description": "Ensures every id attribute value of active elements is unique",
      "help": "IDs of active elements must be unique",
      "helpUrl": "https://dequeuniversity.com/rules/axe/3.5/duplicate-id-active?application=axeAPI",
      "nodes": [
        {
          "any": [
            {
              "id": "duplicate-id-active",
              "data": "fail",
              "relatedNodes": [
                {
                  "html": "<button id=\"fail\">hello</button>",
                  "target": [
                    "button:nth-child(2)"
                  ]
                }
              ],
              "impact": "serious",
              "message": "Document has active elements with the same id attribute: fail"
            }
          ],
          "all": [],
          "none": [],
          "impact": "serious",
          "html": "<button id=\"fail\">hi</button>",
          "target": [
            "button:nth-child(1)"
          ],
          "failureSummary": "Fix any of the following:\n  Document has active elements with the same id attribute: fail"
        }
      ]
    }
  
```

**Comment by connorjclark** [comment]
This is great, sorry for the noise!
